engine/src/flutter/ci/licenses_golden/excluded_files

```diff
@@ -417,6 +417,7 @@
 ../../../flutter/shell/platform/windows/direct_manipulation_unittests.cc
 ../../../flutter/shell/platform/windows/dpi_utils_unittests.cc
 ../../../flutter/shell/platform/windows/fixtures
+../../../flutter/shell/platform/windows/flutter_host_window_controller_unittests.cc
```

Comment:
Looks like the license files need to be updated. See: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8709971693830947249/+/u/test:_licenses_check/stdout